src/renderer/components/dock/create-resource.tsx

```diff
@@ -30,11 +67,26 @@ export class CreateResource extends React.Component<Props> {
     return createResourceStore.getData(this.tabId);
   }
 
+  // templatesStores - provides list of folders with templates in priority order (first overrides others)
+  get templatesStores(): string[] {
+    const userStore = path.join((app || remote.app).getPath("userData"), "templates");
```

Comment:
```suggestion
    const templatesBaseDir = path.join((app || remote.app).getPath("userData"), "templates");
```

Reply:
I think that using `.k8slens/templates` would be better